app/src/main/java/com/team41/boromi/BookActivity.java

```diff
@@ -73,4 +80,24 @@ public void onTabReselected(Tab tab) {
       }
     });
   }
+
+  @Override
+  public boolean onCreateOptionsMenu(Menu menu) {
```

Comment:
due to crunch time, I'm gonna trust you this works andrew

Reply:
yes